<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="36510" started-at="2017-01-03T13:25:34Z" finished-at="2017-01-03T13:26:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="36510" started-at="2017-01-03T13:25:34Z" finished-at="2017-01-03T13:26:11Z">
      <class name="com.pack.home.TestExecution.TestRun">
        <test-method status="PASS" signature="executingTEP()[pri:0, instance:com.pack.home.TestExecution.TestRun@4cc0edeb]" name="executingTEP" is-config="true" duration-ms="34423" started-at="2017-01-03T18:55:34Z" finished-at="2017-01-03T18:56:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- executingTEP -->
        <test-method status="FAIL" signature="login()[pri:0, instance:com.pack.home.TestExecution.TestRun@4cc0edeb]" name="login" duration-ms="1016" started-at="2017-01-03T18:56:09Z" finished-at="2017-01-03T18:56:10Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[C:\NewProject-2017-Maven\test\src\com\pack\home\ObjectRepositories\login.properties (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: C:\NewProject-2017-Maven\test\src\com\pack\home\ObjectRepositories\login.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.pack.home.ObjectRepositories.ExtractingDataFromPropertiesFile.extractData(ExtractingDataFromPropertiesFile.java:9)
	at com.pack.home.Libraries.LibraryForGenericFunction.waitAndSendKeysForID(LibraryForGenericFunction.java:99)
	at com.pack.home.TestExecution.TestRun.login(TestRun.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="loginclick()[pri:0, instance:com.pack.home.TestExecution.TestRun@4cc0edeb]" name="loginclick" duration-ms="1005" started-at="2017-01-03T18:56:10Z" finished-at="2017-01-03T18:56:11Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[C:\NewProject-2017-Maven\test\src\com\pack\home\ObjectRepositories\login.properties (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: C:\NewProject-2017-Maven\test\src\com\pack\home\ObjectRepositories\login.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at com.pack.home.ObjectRepositories.ExtractingDataFromPropertiesFile.extractData(ExtractingDataFromPropertiesFile.java:9)
	at com.pack.home.Libraries.LibraryForGenericFunction.waitAndClickForID(LibraryForGenericFunction.java:83)
	at com.pack.home.TestExecution.TestRun.loginclick(TestRun.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginclick -->
      </class> <!-- com.pack.home.TestExecution.TestRun -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
